pyproject.toml

```diff
@@ -17,6 +17,7 @@ dependencies = [
     "pyzmq >=25.0.0",
     "geventhttpclient >=2.0.11",
     "ConfigArgParse >=1.5.5",
+    "toml >=0.10.2",
```

Comment:
Is the toml package really needed for configargparse to read toml files? The `toml` package look really old and unmaintained. If it is only needed to generate that test file you can just use a static text instead.

Reply:
Yes, [toml](https://github.com/uiri/toml) is required for the configargparse TOMLParser to function. Although Python version 3.11 and later includes a built-in TOML library, it appears that configargparse is not actively maintained. If we want to avoid using toml, I can create our own TOMLParser, which will require [tomli](https://github.com/hukkin/tomli?tab=readme-ov-file#building-a-tomlitomllib-compatibility-layer) for Python versions earlier than 3.11.
<img width="773" alt="Screenshot 2024-02-25 at 11 04 12" src="https://github.com/locustio/locust/assets/87263895/61ac65a5-4179-4fad-9766-b40d26f972bb">
<img width="888" alt="Screenshot 2024-02-25 at 11 55 38" src="https://github.com/locustio/locust/assets/87263895/6a9b4fc1-5751-4325-8716-39fa9f6555ba">

Reply:
That sounds good, please do!